src/libraries/Common/src/Interop/Browser/Interop.Runtime.cs

```diff
@@ -0,0 +1,625 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+
+using JSObject = Interop.JavaScript.JSObject;
+using JSException = Interop.JavaScript.JSException;
+
+internal static partial class Interop
+{
+    internal static partial class Runtime
+    {
+        [MethodImplAttribute(MethodImplOptions.InternalCall)]
+        internal static extern string InvokeJS(string str, out int exceptional_result);
```

Comment:
parameter names should follow the naming conventions

Reply:
Addressed in recent commit